README.md

```diff
@@ -162,6 +162,16 @@ modifiers:
   unichar:       0.1 # Replaces a random consecutive repeated letter with a single letter. 
 ````
 
+#### Noise
+Augments the training data with sentences that are pure nonsensitcal noise. They are identical on the source and target sides and teach the model to learn to deal with unknown input.
+```yaml
+modifiers:
+    - Noise: 0.01
+      min_word_legnth: 2 # Minumum lenght of each fake word, defaults is 2 if left blank
+      max_word_length: 5 # Maximum lenght of each fake word, defaults is 5 if left blank
+      max_words: 4       # Maximu number of fake words, default is 4 if left blank
+```
```

Comment:
The implementation of this modifier `line_in` to `line_in\nRANDOMNOISE` which means a 'line' contains more tsv columns than other modifiers may be anticipating, therefore we should to note that this really must be the last modifier

Reply:
yeah, we discussed it with @jelmervdl that's not the way to do it most likely.